tls/s2n_connection.c

```diff
@@ -747,20 +747,6 @@ int s2n_connection_get_client_cert_chain(struct s2n_connection *conn, uint8_t **
     return 0;
 }
 
-int s2n_connection_get_cipher_preferences(struct s2n_connection *conn, const struct s2n_cipher_preferences **cipher_preferences)
```

Comment:
I moved this to s2n_cipher_preferences.c because its logic now relies on the structure of s2n_cipher_preferences_set.